<launch>
<arg name="port" default="/dev/ttyUSB0" />
<node name="driver" pkg="gps_driver" type="driver.py" output="screen"/>
<param name="gps_port" value="$(arg port)"/>
</launch>
